lib/kamal/cli/templates/deploy.yml

```diff
@@ -18,6 +18,8 @@ servers:
 proxy:
   ssl: true
   host: app.example.com
+  # Change the default application port from 80 to 3000. Useful for non-Thruster apps.
```

Comment:
```suggestion
  # kamal-proxy connects to your container over port 80, use `app_port` to specify a different port.
```

It looks like kamal-site has updated docs for this as well in the kamal-2 branch https://github.com/basecamp/kamal-site/blob/kamal-2/docs/configuration/proxy.md#app-port